realm-jni/build.gradle

```diff
@@ -1,4 +1,4 @@
-ext.coreVersion = '0.89.2'
+ext.coreVersion = '0.89.5'
```

Comment:
Yes I agree. But it's also a hard dependency so I decided to go for it since it's a one liner